tests/Integration/Endpoints/AnalyticsPostsProxyEndpointTest.php

```diff
@@ -228,7 +229,11 @@ public function test_get_items(): void {
 			function () use ( &$dispatched ) {
 				$dispatched++;
 				return array(
-					'body' => '{"data":[
+					'response' => array(
+						'code'    => 200,
+						'message' => 'OK',
+					),
+					'body'     => '{"data":[
```

Comment:
These changes can be removed based on the decision of above comment.

Reply:
Removed.